### PR TITLE
Pools v1.1

### PR DESCRIPTION
**Pull request by Steven-Yulong-Yan**
Group the pools according to the price feed (market) that they track.

Pending: 24h change, new commit modal, 'i' information

### CODE REVIEW

**Comment by Steven-Yulong-Yan** [comment]
We can probably merge this first as it is and add those pending items in separate PRs

File: archetypes/Pools/index.tsx [hunk 1]
```diff
@@ -102,22 +129,70 @@ export const Browse: React.FC = () => {
...
+                            key={index}
+                            className="p-4 md:p-8 lg:px-16 mb-4 shadow-xl rounded sm:rounded-2xl md:rounded-3xl bg-theme-background"
+                        >
+                            <div className="w-full h-20 my-5 p-4 flex justify-between divide-x-4 divide-cool-gray-200 dark:divide-cool-gray-900 overflow-x-auto whitespace-nowrap rounded-xl bg-cool-gray-50 dark:bg-theme-background-secondary">
+                                <div className="flex pr-10">
+                                    <Logo
+                                        className="inline mr-3 my-auto"
+                                        size="lg"
+                                        ticker={key.split('/')[0] as LogoTicker}
+                                    />
```
Comment (dospore) [inline archetypes/Pools/index.tsx:151]: Might need to reduce the size of this logo down a little. Designs have it as 32px.

figma.com/file/Gh6WzO8gLZWDmTvRbw8ikm/Tracer-Style-Guide-%26-Components?node-id=5993%3A204602

File: archetypes/Pools/index.tsx [hunk 2]
```diff
@@ -102,22 +129,70 @@ export const Browse: React.FC = () => {
...
+                </section>
+                {!sortedFilteredTokens.length ? <Loading className="w-10 mx-auto mt-10" /> : null}
+                {Object.keys(groupedSortedFilteredTokens).map((key, index) => {
+                    const dataRows = groupedSortedFilteredTokens[key as any] as BrowseTableRowData[];
+                    return (
+                        <div
+                            key={index}
+                            className="p-4 md:p-8 lg:px-16 mb-4 shadow-xl rounded sm:rounded-2xl md:rounded-3xl bg-theme-background"
+                        >
+                            <div className="w-full h-20 my-5 p-4 flex justify-between divide-x-4 divide-cool-gray-200 dark:divide-cool-gray-900 overflow-x-auto whitespace-nowrap rounded-xl bg-cool-gray-50 dark:bg-theme-background-secondary">
```
Comment (dospore) [inline archetypes/Pools/index.tsx:145]: Designers arent keeping consistent to widths and stuff but the borders are meant to be 3px. I think 4px is fine.

Comment (dospore) [reply archetypes/Pools/index.tsx:145]: I will bump them on remaining consistent to tailwind

File: archetypes/Pools/index.tsx [hunk 3]
```diff
@@ -102,22 +129,70 @@ export const Browse: React.FC = () => {
...
+                                        ticker={key.split('/')[0] as LogoTicker}
+                                    />
+                                    <div className="my-auto">
+                                        <div className="font-bold text-lg">{key}</div>
+                                    </div>
+                                </div>
+                                <div className="px-10">
+                                    <div className="text-cool-gray-500 dark:text-cool-gray-400 font-semibold">
+                                        SPOT PRICE
+                                    </div>
```
Comment (dospore) [inline archetypes/Pools/index.tsx:159]: These have Source Sans Pro font family

File: archetypes/Pools/index.tsx [hunk 4]
```diff
@@ -102,22 +129,70 @@ export const Browse: React.FC = () => {
...
+                                    <a
+                                        href={`https://reputation.link/contracts/${MarketToOracleMap(key)}`}
+                                        target="_blank"
+                                        rel="noreferrer"
+                                        className="flex items-center"
+                                    >
+                                        <img className="mr-2" src={'/img/general/chainlink.svg'} alt="Chainlink" />
+                                        <div className="mr-2 font-bold">Chainlink</div>
+                                        <img src={'/img/general/link.svg'} alt="Link" />
```
Comment (dospore) [inline archetypes/Pools/index.tsx:174]: If you import the image at the top it should change color when the font color changes
image

so do 
`import LinkIcon from 'whatever/path/to/img'`
and use it like
`<LinkIcon alt="Link"/>`

File: archetypes/Pools/FilterSelects/Bar.tsx [hunk 5]
```diff
@@ -20,6 +21,13 @@ const REBALANCE_OPTIONS = [
         key: RebalanceEnum.last,
         text: 'Last Rebalance',
     },
+    {
+        key: RebalanceEnum.historic,
+        text: 'Historic Data',
+        disabled: {
+            optionKey: TooltipKeys.ComingSoon,
+        },
+    },
```
Comment (dospore) [inline archetypes/Pools/FilterSelects/Bar.tsx:30]: The background of this state needs a little update on dark mode 
image

Comment (Steven-Yulong-Yan) [reply archetypes/Pools/FilterSelects/Bar.tsx:30]: ✅

File: archetypes/Pools/index.tsx (hunk 4 above)
Comment (Steven-Yulong-Yan) [reply archetypes/Pools/index.tsx:174]: ✅

File: archetypes/Pools/index.tsx (hunk 3 above)
Comment (Steven-Yulong-Yan) [reply archetypes/Pools/index.tsx:159]: You raised this in the chat so I'll leave this until the designers respond.

File: archetypes/Pools/index.tsx (hunk 2 above)
Comment (Steven-Yulong-Yan) [reply archetypes/Pools/index.tsx:145]: Updated to 3px

File: archetypes/Pools/index.tsx (hunk 1 above)
Comment (Steven-Yulong-Yan) [reply archetypes/Pools/index.tsx:151]: Updated to 32px

**Review by dospore — APPROVED** [review]

**Pull request merged**